### PR TITLE
Update GitHub Action documentation

### PR DESCRIPTION
**Pull request by phracek**
This pull request only updates README.md file
and add to .gitignore `.idea` folder

Signed-off-by: Petr "Stone" Hracek <phracek@redhat.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,12 +1,74 @@
-# About
-GitHub Action to execute tests by Testing Farm and update Pull Request status.
+# testing-farm-as-github-action
 
-The Action uses ubuntu-20.04
+Testing Farm as GitHub Action is a GitHub Action to execute tests by Testing Farm and update Pull Request status.
```
Comment (zmiklank) [inline README.md:3]:
```suggestion
Testing Farm as GitHub Action is a GitHub Action to execute tests by Testing Farm and update Pull Request status.
```
Testing Farm as a GitHub Action is a GitHub Action for executing tests against specific Pull Requests on the Testing Farm infrastructure. The GitHub action reports the result of the tests with Pull Request status.

File: README.md [hunk 2]
```diff
@@ -1,12 +1,74 @@
...
 
-## Prerequisities
+This action only runs on `Linux`. Before a calling this GitHub Action, you must first clone your project by e.g.
```
Comment (zmiklank) [inline README.md:5]:
```suggestion
This action internally runs on ubuntu-20.04. Before calling this GitHub Action, you must first clone your project by, e.g., 
```
I have checked the code, and it does not seem, that the user can choose the base Linux distribution for this action. I would say, that is sufficient to state, that the action runs on ubuntu.

File: README.md [hunk 3]
```diff
@@ -1,12 +1,74 @@
-# About
-GitHub Action to execute tests by Testing Farm and update Pull Request status.
+# testing-farm-as-github-action
 
-The Action uses ubuntu-20.04
+Testing Farm as GitHub Action is a GitHub Action to execute tests by Testing Farm and update Pull Request status.
 
-## Prerequisities
+This action only runs on `Linux`. Before a calling this GitHub Action, you must first clone your project by e.g. 
+by GitHub Action [Checkout V2](https://github.com/actions/checkout).
```
Comment (zmiklank) [inline README.md:6]:
```suggestion
GitHub Action [Checkout V2](https://github.com/actions/checkout).
```

File: README.md (hunk 2 above)
Comment (zmiklank) [reply README.md:5]: Interesting, good to know, thanks :).

File: README.md [hunk 4]
```diff
@@ -1,12 +1,74 @@
...
+ 
+In your organization secrets you should have stored API key to testing farm.
+
+## Action Inputs
+
+| Input Name      | Description             | Default |
+|-----------------|-------------------------| ------- |
+| api_key         | A testing farm API key. | **Must be provided** |  
+| tmt_repository | An url to tmt repository | **Must be provided** |
+| test_fmf_plan | A fmf plan which will be selected. | All plans |
```
Comment (zmiklank) [inline README.md:21]: What means "All plans"? It is not clear from this table.

Comment (phracek) [reply README.md:21]: It means all plans in fmf plan.

File: README.md (hunk 2 above)
Comment (phracek) [reply README.md]: Fixed by https://github.com/sclorg/testing-farm-as-github-action/pull/6

**Pull request merged**